### DISCUSSION TITLE
restarting a simulation

### DISCUSSION THREAD

**Discussion by dineshkumarreddyg36** [xcompact3d/Incompact3d#102] [Q&A] [answered]
I wish to restart a simulation. There are some restart files written, say the restart file name is '**restart0003000.info**'. I want to restart my simulation from here. what value should I have to give for the variable '**irestart**' in the input file?

**Comment by mathrack** [comment, marked as answer]
Hello,

I think the restart is not the `restart0003000.info` file but the `restart0003000` one.

If the last time step of your simulation was `ilast = 3000`. Then you should restart with `ifirst = 3001` and `irestart = 1`.

**Reply by dineshkumarreddyg36** [reply]
okay, thank you @mathrack

**Comment by pbartholomew08** [comment]
@mathrack is correct, the restart data is stored in the checkpoint file, to avoid filling a file quota with (potentially large) checkpoint files the checkpoint is overwritten, there is no checkpoint history. Every time it is overwritten a new restartXXXXX.info is written to describe the file.

You can only restart from your latest checkpoint time, if that was at t=3000 you should restart at time t=3001 using `ifirst=3001` and `irestart=1`

**Reply by dineshkumarreddyg36** [reply]
got it, thank you @pbartholomew08
